samples/snippets/deployment_and_config_test.py

```diff
@@ -36,12 +38,37 @@
 
 PROJECT_ID = "python-docs-samples-tests"
 CONFIG_ID = "my-game-server-config"
-REALM_LOCATION = "us-central1"
+REALM_LOCATION = "global"
+
+# The format of realm ID. This is used in the unit tests and cleanup below.
+realm_id_format = 'test-realm-{}-{}'
+
+# The format of deployment ID. This is used in the unit tests and cleanup below.
+deployment_id_format = 'test-deployment-{}-{}'
+
+
+@pytest.fixture(scope="session", autouse=True)
+def clean_up_old_deployments():
+    all_deployments = list_deployments.list_deployments(PROJECT_ID)
+    for deployment in all_deployments:
+        deployment_name = deployment.name
+        deployment_id = deployment_name[deployment_name.rfind('/') + 1: len(deployment_name)]
+        if deployment_id.find('test-deployment-') == 0:
+            time_str = deployment_id[deployment_id.rfind('-') + 1: len(deployment_id)]
+            test_date = datetime.datetime.utcfromtimestamp(int(time_str))
+            now_date = datetime.datetime.utcfromtimestamp(int(time.time()))
+            difftime = now_date - test_date
+
+            # *NOTE* Restrict to deployments used in the tests older than 7 days
+            #        to prevent thrashing in the case of async tests
+            if (difftime.days > 7):
```

Comment:
7 days seems long, perhaps 1 or 2 days?

Reply:
Thanks for reviewing.
Changed to 2 days. I want to leave enough time for debugging if there are issues.